takwimu/dataset_release_utils.py

```diff
@@ -0,0 +1,51 @@
+from django.conf import settings
+
+
+def get_country_geo_code(geo):
+    level = geo.geo_level.lower()
+    country = geo.geo_code \
+        if level == 'country' \
+        else geo.ancestors()[-1].geo_code
```

Comment:
It is possible to navigate to continent level in Takwimu. This doesn't handle that.

![screenshot from 2018-12-13 14-12-35](https://user-images.githubusercontent.com/1779590/49935691-c373a480-fee2-11e8-98ac-ce25be2866ed.png)

Reply:
@kilemensi I think we should redirect https://takwimu.africa/profiles/continent-AFR-africa/ to https://takwimu.africa/profiles/

Reply:
As a last resort, yes we can do that but why not handle it graceful in code first @phillipahereza?

Reply:
@kilemensi but we have no data at continent level

Reply:
Yes @phillipahereza. It should be an empty page and not an error. We can then *choose* to redirect.